plenum/common/exceptions.py

```diff
@@ -229,4 +229,17 @@ class OrientDBNotRunning(GraphStorageNotAvailable):
 
 
 class InvalidEndpoint(Exception):
-    pass
\ No newline at end of file
+    def __init__(self, endpoint):
+        super().__init__('invalid endpoint: {}'.format(endpoint))
+
+
+class InvalidEndpointMissing(InvalidEndpoint):
```

Comment:
Confused by this error's name

two meanings come to mind:
1. The endpoint is missing (this is the one I believe is what is meant)
2. The missing endpoint is invalid (but this one don't make much sense.

I suggest we rename it to MissingEndpoint if #1 is correct.